Sources/_InternalTestSupport/Helpers.swift

```diff
@@ -84,42 +85,46 @@ public let swiftc: AbsolutePath = {
     return swiftcPath
 }()
 
-public let lldb: AbsolutePath = {
-    if let environmentPath = ProcessInfo.processInfo.environment["LLDB_PATH"] {
-        return try! AbsolutePath(validating: environmentPath)
-    }
+// public let lldb: AbsolutePath = {
+//     if let environmentPath = ProcessInfo.processInfo.environment["LLDB_PATH"] {
+//         return try! AbsolutePath(validating: environmentPath)
+//     }
 
-    // We check if it exists because lldb doesn't exist in Xcode's default toolchain.
-    let toolchainLLDBPath = toolchainPath.appending(components: "usr", "bin", "lldb")
-    if localFileSystem.exists(toolchainLLDBPath) {
-        return toolchainLLDBPath
-    }
+//     // We check if it exists because lldb doesn't exist in Xcode's default toolchain.
+//     let toolchainLLDBPath = toolchainPath.appending(components: "usr", "bin", "lldb")
+//     if localFileSystem.exists(toolchainLLDBPath) {
+//         return toolchainLLDBPath
+//     }
 
-    #if os(macOS)
-    let lldbPath = try! AbsolutePath(
-        validating: sh("xcrun", "--find", "lldb").stdout.spm_chomp()
-    )
-    return lldbPath
-    #else
-    fatalError("LLDB_PATH environment variable required")
-    #endif
-}()
+//     #if os(macOS)
+//     let lldbPath = try! AbsolutePath(
+//         validating: sh("xcrun", "--find", "lldb").stdout.spm_chomp()
+//     )
+//     return lldbPath
+//     #else
+//     fatalError("LLDB_PATH environment variable required")
+//     #endif
+// }()
 
-public let swiftpmBinaryDirectory: AbsolutePath = {
-    if let environmentPath = ProcessInfo.processInfo.environment["SWIFTPM_BIN_DIR"] {
-        return try! AbsolutePath(validating: environmentPath)
-    }
 
-    return swift.parentDirectory
-}()
 
-public let swiftBuild: AbsolutePath = swiftpmBinaryDirectory.appending(component: "swift-build")
+// public let swiftpmBinaryDirectory: AbsolutePath = {
```

Comment:
chore (blocking): remove lie 111 to 127. They should be replaced with the `executeSwift*` functions.